iambic/core/models.py

```diff
@@ -310,6 +310,7 @@ class ProposedChangeType(Enum):
     DELETE = "Delete"
     ATTACH = "Attach"
     DETACH = "Detach"
+    UNKNOWN = "Unknown"
```

Comment:
What is UNKNOWN for? Maybe needs a comment?

Reply:
Yes, I added it when an exception is raised during the apply method at AwsTemplate. The variable exception_seen should be a ProposedChanged but the code assumed that it was an string.